tests/Unit/Subscriptions/Storage/RedisStorageManagerTest.php

```diff
@@ -94,11 +94,11 @@ public function testDeleteSubscriber(): void
         $subscriber = new DummySubscriber($channel, 'test-topic');
         $redisConnection->expects($this->exactly(3))
             ->method('command')
-            ->with(...$this->withConsecutive(
+            ->with(...array_values([...$this->withConsecutive(
```

Comment:
```suggestion
            ->with(...array_values($this->withConsecutive(
```

spreading once should suffice